config/locales/en.yml

```diff
@@ -162,13 +219,35 @@ en:
         add: Add
         update: Update
         current_admins:
-          actions: Action
-          email: Email address
           header: Current Account Administrators
+          all_users: All Users
```

Comment:
remove line 223-225 for this and all other languages

Reply:
Removed lines 223-225 for all languages.

Reply:
Removed lines 223-225 for all languages.